app/models/menu.rb

```diff
@@ -1,5 +1,5 @@
 class Menu < ActiveRecord::Base
-  has_and_belongs_to_many :menu_categories
+  has_and_belongs_to_many :menu_items
```

Comment:
Yes, in other words every record in the join table is unique by both menu_id and menu_item_id.